css/style.css

```diff
@@ -0,0 +1,8 @@
+.lentele{
+    background-color: #2820ff;
+    color: white;
+}
+.lentele1{
+    background-color: #51ffab;
+    color: grey;
+}
```

Comment:
https://mrkiffie.com/2016/newlines-and-version-control/
Gali paskaityti arba tiesiog prideti nauja eilute failo gale